examples/tv-casting-app/android/App/app/src/main/jni/com/chip/casting/TvCastingApp.java

```diff
@@ -29,6 +29,46 @@ public native boolean openBasicCommissioningWindow(
 
   public native boolean sendUserDirectedCommissioningRequest(String address, int port);
 
+  /**
+   * Send a request to Discover commissioners using DNS-SD
+   *
+   * @param discoverySuccessCallback
+   * @param discoveryFailureCallback
+   * @return true, if the discovery request was sent successfully, false otherwise
+   */
+  public boolean discoverCommissioners(
```

Comment:
Will this discover any commissioner or only video players? If only video players does it make sense to rename this to discoverVideoPlayers?

Reply:
Sure, I'll update the name to discoverVideoPlayers